Add link and note to exported csv

yabirgb opened the pull request:
Closes #2891

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

yabirgb commented:
Forgot to review the tests for accounting. Will check

LefterisJP commented on rotkehlchen/csv_exporter.py:
```diff
@@ -53,6 +53,7 @@
 FILENAME_DEFI_EVENTS_CSV = 'defi_events.csv'
 FILENAME_LEDGER_ACTIONS_CSV = 'ledger_actions.csv'
 FILENAME_ALL_CSV = 'all_events.csv'
+ETH_EXPLORER = 'https://etherscan.io/tx/'
```
you also have this in accountant.py. Choose one.

LefterisJP reviewed (commented):
Generally looks okay. Left 2 comments.

LefterisJP reviewed (commented):
The frontend check seems to have failed

kelsos commented:
All the e2e tests failed... also I need to change the setup for the backend logs to make sure they are uploaded on failure

yabirgb commented:
Fixed the issue with e2e and also changed the CI to upload the .log file from the backend. About the failing tests in the backend is a bad response from blockstream, it works when I executed it locally

LefterisJP commented on rotkehlchen/csv_exporter.py:
```diff
@@ -103,6 +105,21 @@ def __init__(
+                self.eth_explorer = frontend_settings['explorers']['ETH']['transaction']
+            else:
+                self.eth_explorer = ETH_EXPLORER
+        except json.decoder.JSONDecodeError:
```
you need to catch a key error here too

yabirgb replied:
I thought about a keyerror but for how is the condition of that `if` it can never happen. Maybe I should have removed that if completely and just catch the error

LefterisJP reviewed (approved):
LGTM. just need to catch another error too. I will add it and let the tests run,

kelsos commented:
From what I can see the failed test was due to a read timeout @yabirgb if you can also confirm I can proceed and merge the PR

yabirgb commented:
> From what I can see the failed test was due to a read timeout @yabirgb if you can also confirm I can proceed and merge the PR

affirmative, timeout from etherscan.
